src/index.html

```diff
@@ -9,10 +12,60 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__bg"></div>
+      <div class="product-card__details">
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h3>
+        <div class="product-card__code">Product code: 195434</div>
+        <div class="product-card__rating">
+          <div class="product-card__stars">
+            <span
+              class="product-card__star"
+              data-value="1"
+            ></span>
+            <span
+              class="product-card__star"
+              data-value="2"
+            ></span>
+            <span
+              class="product-card__star"
+              data-value="3"
+            ></span>
+            <span
+              class="product-card__star"
+              data-value="4"
+            ></span>
+            <span
+              class="product-card__star"
+              data-value="5"
+            ></span>
+          </div>
+          <div class="product-card__revievs">Reviews: 5</div>
+        </div>
+        <div class="product-card__price">
+          <div class="product-card__price-title">Price:</div>
+          <div class="product-card__price-sum">$2,199</div>
+        </div>
+        <button
+          type="button"
+          href="#"
```

Comment:
The `href` attribute is not valid for a `button` element. If you want to use a button to navigate, consider using a JavaScript event listener to handle the navigation.